metrics/sinks/poseidon/driver.go

```diff
@@ -0,0 +1,114 @@
+package poseidon
+
+import (
+	"github.com/ICGog/poseidongo/pkg/stats"
+	"golang.org/x/net/context"
+	"google.golang.org/grpc"
+	"k8s.io/heapster/metrics/core"
+	"log"
+	"net/url"
+	"sync"
+)
+
+type PoseidonSink struct {
+	stats.PoseidonStatsClient
+	sync.RWMutex
+}
+
+func (sink *PoseidonSink) Name() string {
+	return "Poseidon Sink"
+}
+
+func (sink *PoseidonSink) Stop() {
+	// Do nothing.
+}
+
+func (sink *PoseidonSink) ExportData(dataBatch *core.DataBatch) {
+	sink.Lock()
+	defer sink.Unlock()
+
+	for metricKey, metricSet := range dataBatch.MetricSets {
+		if isNodeMetric(metricKey, metricSet) {
+			nodeStats := stats.NodeStats{
+				Hostname:       metricSet.Labels[core.LabelHostname.Key],
+				CpuAllocatable: getMetricValue(metricSet, core.MetricNodeCpuAllocatable).FloatValue,
```

Comment:
Can you please double check if all the fields the fields are floats and not uint64? In our gRPC service they are uint64 because that's what I was getting back from the K8s api server.